src/cli/commands/force/lightning/lwc/test/utam/run.ts

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { flags, SfdxCommand } from '@salesforce/command';
+import { Logger, Messages, SfError } from '@salesforce/core';
+import { AndroidEnvironmentRequirements } from '@salesforce/lwc-dev-mobile-core/lib/common/AndroidEnvironmentRequirements';
+import {
+    CommandLineUtils,
+    FlagsConfigType
+} from '@salesforce/lwc-dev-mobile-core/lib/common/Common';
+import { CommonUtils } from '@salesforce/lwc-dev-mobile-core/lib/common/CommonUtils';
+import { IOSEnvironmentRequirements } from '@salesforce/lwc-dev-mobile-core/lib/common/IOSEnvironmentRequirements';
+import {
+    RequirementProcessor,
+    HasRequirements,
+    CommandRequirements
+} from '@salesforce/lwc-dev-mobile-core/lib/common/Requirements';
+import util from 'util';
+import fs from 'fs';
+import path from 'path';
+
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = Messages.loadMessages(
+    '@salesforce/lwc-dev-mobile',
+    'test-utam-run'
+);
+
+export class Run extends SfdxCommand implements HasRequirements {
+    public static description = messages.getMessage('commandDescription');
+
+    public static examples = [
+        `$ force:lightning:lwc:test:utam:run -p iOS --config '/path/to/wdio.conf.js' --spec '/path/to/myTest.spec.js'`,
+        `$ force:lightning:lwc:test:utam:run -p Android --config '/path/to/wdio.conf.js' --spec '/path/to/myTest.spec.js'`
+    ];
+
+    public static cmd = `npx wdio --version`;
+
+    private static createError(stringId: string): SfError {
+        return new SfError(
+            messages.getMessage(stringId),
+            'lwc-dev-mobile',
+            Run.examples
+        );
+    }
+
+    public static flagsConfig = {
+        ...CommandLineUtils.createFlagConfig(FlagsConfigType.Platform, true),
+        config: flags.string({
+            description: messages.getMessage('configFlagDescription'),
+            required: true,
+            validate: (wdioConfig) => {
```

Comment:
In the latest changes to `UTAM` branch, I updated the `Configure` which you can refer to as an example. But basically there is no need to check if a flag has non-empty value since we're extending from `SfdxCommand` which checks this already.